Projects/Claims/Sources/Views/ClaimDetailView.swift

```diff
@@ -167,6 +77,175 @@ public struct ClaimDetailView: View {
         }
     }
 
+    @ViewBuilder
+    private var infoAndContactSection: some View {
+        hSection {
+            hRow {
+                hText(statusParagraph)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .multilineTextAlignment(.leading)
+            }
+            hRow {
+                ContactChatView(
+                    store: vm.store,
+                    id: vm.claim.id,
+                    status: vm.claim.status.rawValue
+                )
+                .padding(.bottom, 4)
+            }
+        }
+    }
+
+    @ViewBuilder
+    private var memberFreeTextSection: some View {
+        if let inputText = vm.claim.memberFreeText {
+            hSection {
+                hRow {
+                    hText(inputText)
+                }
+            }
+            .withHeader {
+                hText(L10n.ClaimStatusDetail.submittedMessage)
+                    .padding(.leading, 2)
+            }
+            .padding(.top, 16)
+        }
+    }
+
+    @ViewBuilder
+    private var claimDetailsSection: some View {
+        VStack(spacing: 16) {
+            hSection {
+                VStack(spacing: 8) {
+                    claimDetailsRow(title: L10n.ClaimStatus.ClaimDetails.type, value: vm.claim.claimType)
+                    if let incidentDate = vm.claim.incidentDate {
+                        claimDetailsRow(
+                            title: L10n.ClaimStatus.ClaimDetails.incidentDate,
+                            value: incidentDate.localDateToDate?.displayDateDDMMMYYYYFormat ?? ""
+                        )
+                    }
+                    if let submitted = vm.claim.submittedAt {
+                        claimDetailsRow(
+                            title: L10n.ClaimStatus.ClaimDetails.submitted,
+                            value: submitted.localDateToIso8601Date?.displayDateDDMMMYYYYFormat ?? ""
+                        )
+                    }
+                }
+            }
+            .withHeader {
+                hText(L10n.ClaimStatus.ClaimDetails.title)
+            }
+            .hWithoutDivider
+            .sectionContainerStyle(.transparent)
+
+            termsAndConditions
+        }
+    }
+
+    @ViewBuilder
+    private func claimDetailsRow(title: String, value: String) -> some View {
+        HStack {
+            hText(title)
+                .foregroundColor(hTextColor.secondary)
+            Spacer()
+            hText(value)
+                .foregroundColor(hTextColor.secondary)
+        }
+    }
+
+    @ViewBuilder
+    private var termsAndConditions: some View {
+        if let termsAndConditionsDocument = vm.claim.productVariant?.documents.first {
```

Comment:
It feels a little bit scary to use first one. Isn't there some better way to get only t&c document?

Reply:
Fixed now!